test/e2e/throughput_test.go

```diff
@@ -6,13 +6,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/celestiaorg/celestia-app/pkg/appconsts"
-	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	"github.com/celestiaorg/celestia-app/v2/pkg/appconsts"
+	"github.com/celestiaorg/celestia-app/v2/test/util/testnode"
 	"github.com/stretchr/testify/require"
 )
 
 func TestE2EThroughput(t *testing.T) {
-	if os.Getenv("KNUU_NAMESPACE") != "test" {
+	if os.Getenv("KNUU_NAMESPACE") != "test-sanaz" {
```

Comment:
[blocking] I think we need to remove the `-sanaz` prefix before merging

Reply:
Of course, please see https://github.com/celestiaorg/celestia-app/pull/3244/commits/821cf940d59ba559a8da78061db95f6129157b2d